Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -18,7 +20,7 @@
 @Singleton
 public class DriveSubsystem extends BaseSubsystem implements TelemetrySource {
     private static Logger log = Logger.getLogger(DriveSubsystem.class);
-
+    public final XGyro imu;
```

Comment:
Given that you're working from the DriveToOrientation branch, we should probably wait for that to get checked in before merging your changes.